src/components/GenderSettings/__tests__/GenderSettings.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import GenderSettings from '../GenderSettings';
+
+describe('COMPONENT - GenderSettings', () => {
+  it('renders GenderSettings correctly with checked male', () => {
+    const { container } = render(<GenderSettings />);
+    const radioButton = container.querySelectorAll('input')[0];
+
+    fireEvent.change(radioButton, { target: { value: 'male' } });
+    expect(radioButton.value).toBe('male');
```

Comment:
I would change this test slightly to check whether it has 'Male' label.
As far as I remember assertion will look like: `expect(radioButton).toHaveTextContent('Male')`